### ISSUE TITLE
Downloads from Xbox Store broken

### ISSUE THREAD

**Issue by troych** [crazy-max/WindowsSpyBlocker#24]
#### Steps to reproduce this issue
1. Turn on winspy on OpenWRT "adblock" package (this downloads data/hosts/win10/spy.txt)
2. Xbox One store does not download games/apps

#### Expected Behaviour
Winspy turned off. 

Downloading Games and Apps on the Xbox One Store works. (Tried with different demos)

![winspyoff](https://cloud.githubusercontent.com/assets/4061369/20403056/8fcb7c20-acff-11e6-915b-805e40046965.jpg)

#### Actual Behaviour

Winspy turned on. 

Downloading a demo does not work, took me a while to trace it back to me turning on winspy on the OpenWRT adblock opkg package. This downloads your list from https://raw.githubusercontent.com/crazy-max/WindowsSpyBlocker/master/data/hosts/win10/spy.txt

![winspyon](https://cloud.githubusercontent.com/assets/4061369/20403047/87c986de-acff-11e6-9bb2-ce207fc86cee.jpg)

#### Configuration

**Country (ex. United-States)** :
Switzerland

**Operating system (ex. Windows 10 Pro 64 bits)** :
Xbox One (Xbox Store)

**Winver screenshot** :
Does not apply

**Comment by troych** [comment]
I am in the process of checking which blocked host actually breaks the store, I was running my own set of blocked windows telemetry before using this one which didn't interfere. Checked for duplicates so right now I can say that these are not causing any problems:

```
( 64 dupe of 60 ): activate.adobe.com
( 75 dupe of 62 ): adobeereg.com
( 76 dupe of 63 ): www.adobeereg.com
( 77 dupe of 60 ): activate.adobe.com
( 78 dupe of 65 ): activate-sea.adobe.com
( 79 dupe of 66 ): activate-sjc0.adobe.com
( 80 dupe of 67 ): wwis-dubc1-vip60.adobe.com
( 81 dupe of 68 ): 192.150.18.108
( 90 dupe of 61 ): practivate.adobe.com
( 92 dupe of 67 ): wwis-dubc1-vip60.adobe.com
( 121 dupe of 35 ): a-0001.a-msedge.net
( 134 dupe of 54 ): a.ads1.msn.com
( 136 dupe of 55 ): a.ads2.msn.com
( 140 dupe of 56 ): adnexus.net
( 141 dupe of 57 ): adnxs.com
( 142 dupe of 51 ): ads.msn.com
( 143 dupe of 52 ): ads1.msads.net
( 144 dupe of 53 ): ads1.msn.com
( 148 dupe of 58 ): az361816.vo.msecnd.net
( 149 dupe of 59 ): az512334.vo.msecnd.net
( 162 dupe of 14 ): choice.microsoft.com
( 163 dupe of 15 ): choice.microsoft.com.nsatc.net
( 164 dupe of 33 ): compatexchange.cloudapp.net
( 166 dupe of 40 ): corp.sts.microsoft.com
( 167 dupe of 32 ): corpext.msitadfs.glbdns2.microsoft.com
( 168 dupe of 34 ): cs1.wpc.v0cdn.net
( 181 dupe of 16 ): df.telemetry.microsoft.com
( 182 dupe of 39 ): diagnostics.support.microsoft.com
( 187 dupe of 46 ): feedback.microsoft-hohm.com
( 188 dupe of 47 ): feedback.search.microsoft.com
( 189 dupe of 45 ): feedback.windows.com
( 196 dupe of 43 ): i1.services.social.microsoft.com
( 197 dupe of 44 ): i1.services.social.microsoft.com.nsatc.net
( 209 dupe of 7 ): oca.telemetry.microsoft.com
( 210 dupe of 8 ): oca.telemetry.microsoft.com.nsatc.net
( 216 dupe of 42 ): pre.footprintpredict.com
( 217 dupe of 49 ): preview.msn.com
( 219 dupe of 48 ): rad.msn.com
( 220 dupe of 13 ): redir.metaservices.microsoft.com
( 221 dupe of 17 ): reports.wes.df.telemetry.microsoft.com
( 225 dupe of 19 ): services.wes.df.telemetry.microsoft.com
( 226 dupe of 26 ): settings-sandbox.data.microsoft.com
( 232 dupe of 20 ): sqm.df.telemetry.microsoft.com
( 233 dupe of 9 ): sqm.telemetry.microsoft.com
( 234 dupe of 10 ): sqm.telemetry.microsoft.com.nsatc.net
( 236 dupe of 28 ): survey.watson.microsoft.com
( 239 dupe of 5 ): telecommand.telemetry.microsoft.com
( 240 dupe of 6 ): telecommand.telemetry.microsoft.com.nsatc.net
( 241 dupe of 23 ): telemetry.appex.bing.net
( 242 dupe of 21 ): telemetry.microsoft.com
( 243 dupe of 24 ): telemetry.urs.microsoft.com
( 253 dupe of 27 ): vortex-sandbox.data.microsoft.com
( 255 dupe of 4 ): vortex-win.data.microsoft.com
( 258 dupe of 3 ): vortex.data.microsoft.com
( 259 dupe of 29 ): watson.live.com
( 260 dupe of 30 ): watson.microsoft.com
( 261 dupe of 22 ): watson.ppe.telemetry.microsoft.com
( 262 dupe of 11 ): watson.telemetry.microsoft.com
( 263 dupe of 12 ): watson.telemetry.microsoft.com.nsatc.net
( 265 dupe of 18 ): wes.df.telemetry.microsoft.com
```

**Comment by troych** [comment]
Mh, now it works, I was testing it a couple of times with this list on/off and could reliably make it fail only when it was on. No idea what's different now. I guess this can be closed, I'll reopen if I can pinpoint the problem.

**Comment by crazy-max** [comment]
Ok keep me in touch!

**Comment by troych** [comment]
Sorry for the spam, the blocklist sorting takes quite some time on my poor underpowered router thats what confused me after testing it too quickly after restarting the service.  Not being able to manually clear the DNS cache on the Xbox doesn't help either.

I could pinpoint it to this entry now: displaycatalog.mp.microsoft.com

Adding/Removing that entry breaks/enables downloads reliably. Restarted the console a couple of times too to make sure I've got no caching issues.

**Comment by crazy-max** [comment]
Ok thanks for your time i'll take a look to your issue asap

**Comment by crazy-max** [comment]
Fixed on 3.7.0
